classes/music_album.rb

```diff
@@ -0,0 +1,18 @@
+require '../item'
+
+class MusicAlbum < Item
+  attr_accessor :on_spotify, :name, :publish_date
+  attr_reader :items
+
+  def initialize(on_spotify, name, publish_date)
+    super(publish_date)
```

Comment:
Please look at the API of the parent class i.e the item class. It has the id, as the first argument followed by the publish date. However in the code, the super class is called with only the publish date